scripts/check-log-style.sh

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# zap field name should be camelCase, excepts for idioms and special terms.
+grep -RnE "zap.[A-Z][a-z]+\(\".*-.*\"" cdc tests pkg |
```

Comment:
`zap.String("node name", r.name)`
This zap filed style (words split by space) is not checked.

Reply:
Addressed, thanks!